### ISSUE TITLE
Internal reviewer comments on Section 2 (Simulation and Mock Galaxy Catalog)

### ISSUE THREAD

**Issue by sschmidt23** [LSSTDESC/PZDC1paper#20]
Line numbers refer to the "internal reviewer" version of the PDF attached to the Confluence page:
https://confluence.slac.stanford.edu/pages/viewpage.action?pageId=238561496

**Comments from Dan Gruen**
- [x] l220: Since the clustering of galaxies is not used anywhere, I wonder why you're using N-body simulations at all. Maybe explain.

- [x] L241: the e^5 notation confuses me; give in powers of 10 (also in l302)

- [x] l270: explain somewhere here what “non-detections” in the catalog are – not looked, or looked but not seen? Codes treat these differently later, so it would be good to know what's the right thing.

- [x] L278: On what basis do you argue that 200M galaxies are 2 order of magnitude too many for testing? It's particularly worrisome that you use a small area of 8 sq .deg., inside which galaxy redshifts and colors are correlated by the structures in the N-body simulations. I would not be surprised by a good fraction of a per-cent error on mean redshift (and similar metrics) due to cosmic variance in your training sample. Is it totally unfeasible to re-do these tests with a larger or, better, the full set of galaxies? It would also save you the hassle of kernel density smoothing for any comparisons later.

- [x] L282: This may be a good place to comment on the low mean z of 0.7 in your resulting sample. DES Y1 has ~0.6, and one would of course hope for a much deeper and consequently higher-z sample from LSST.

- [x] 2.1.3: this would be another good place (besides the introduction) for explaining that this sort of the easiest of null tests.

**Comments from Michael Troxel**
- [x] 241: Do 5e5 SDSS galaxies produce a representative training sample for LSST? What tests were done to demonstrate this?

- [x] 278: I know that this number matches the requirement in the SRM, but how did you demonstrate you only needed 1e6 objects for these tests (actually, does it meet the SRM requirement for the ‘gold sample’ in this paper, which is smaller)? Need to provide details, and justify that cosmic variance won’t impact your tests (ie, why choose 8 sq deg when you could subsample 400). Also, the resulting ‘training set’ after cuts is only 4e4 objects large, which is much smaller than the size of the input SDSS training data - is this an issue? Are there really only 4e4 galaxy ‘types’ in the LSST gold sample?

 
**Comments from Markus Michael Rau**
- [x] 276: I don't understand why you select a 8 sq. deg. field instead of downsampling the full area. 8 sq. deg. isn't too small, so it may not be an issue, but doesn't this run the risk of being cosmic variance affected due to structure in the density field? In contrast if you just downsample you are only dealing with statistical noise, for which estimators have well-defined error properties, e.g. ~ 1/sqrt(N).

- [x] 280: Which systematic problems?

- [x] 305: I would give more details on how you tune and perform the K-mean clustering analysis. It seems a pretty important part of the analysis but is merely mentioned here

- [x] 320: Do you think EASY has an unfair advantage/disadvantage if it is run slightly differently to the other codes?
